persistent/Database/Persist/Types.hs

```diff
@@ -60,6 +62,7 @@ import Database.Persist.Types.Base
        , FieldCascade(..)
        , FieldDef(..)
        , FieldType(..)
+       , FieldTypeLit(..)
```

Comment:
Given the comment at the top here:

https://github.com/yesodweb/persistent/blob/77545dc8a6a55c66ff6ed476175684c00943f8f4/persistent/Database/Persist/Types.hs#L48-L50

I wasn't sure if this was the right place for this import, but seemed like it was where everything else was re-exported (I need to use this type in `Persist/TH.hs` I think...)

Going forward should I import this from `Database.Persist.Types.Base` ? Or is re-exporting it here correct for the time being?

Reply:
I'd rather not expose data constructors to the end-user in a non `Internal` module, since any change to those constructors necessitates a breaking change to the API version.

Reply:
In this case, I'd be fine exporting it from `Database.Persist.Quasi.Internal`, since that's where it is used.